stdlib/3/multiprocessing/__init__.pyi

```diff
@@ -74,6 +92,13 @@ class Value():
     value: Any = ...
     def __init__(self, typecode_or_type: str, *args: Any, lock: bool = ...) -> None: ...
 
+class Array():
+    def __init__(self,
+                 typecode_or_type: str,
+                 size_or_intializer: Union[int, Sequence[Any]],
+                 *args: Any,
```

Comment:
Why the `*args`? The signature I see in 3.6 is `def Array(self, typecode_or_type, size_or_initializer, *, lock=True):`.

Also, ideally this class should be in `sharedctypes.pyi`, and it doesn't appear to really be a class.